providers/pool.rb

```diff
@@ -188,7 +188,7 @@ def configure
     configure_application_pool(is_new_auto_start, "autoStart:#{new_resource.auto_start}")
```

Comment:
For some reason I don't seem to be able to add a note at the correct line - but line 34 in this file also does the capitalize thing.  It does it correctly - so it shouldn't matter but if you're setting it here, you might want to DRY and remove those lines as well :)